src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthorizationBackend.java

```diff
@@ -99,7 +99,7 @@ public class LDAPAuthorizationBackend implements AuthorizationBackend {
     private final Settings settings;
     private final WildcardMatcher skipUsersMatcher;
     private final WildcardMatcher nestedRoleMatcher;
-
+    //private final boolean followReferrals;
```

Comment:
Delete commented out code

Reply:
In a couple of places

Reply:
Sorry, I am not sure why we are removing the shouldFollowReferrals attribute here. Since search is called from the AuthorizationBackend file, I was under the impression it was necessary to be able to specify the referral following behavior on these searches as well.

Perhaps I am mistaken though so just let me know.